FR/asciidoc/src/08_concepts.adoc

```diff
@@ -1,18 +1,18 @@
 ifndef::imagesdir[:imagesdir: ../images]
 
 [[section-concepts]]
-== Concepts transverses
+== Concepts transversaux
 
 
 [role="arc42help"]
 ****
 .Contenu
-Cette section décrit les réglementations générales, principales et les idées de solutions qui sont pertinentes dans plusieurs parties (= transversales) de votre système.
+Cette section décrit les réglementations générales, principales et les idées de solutions qui sont importantes dans plusieurs parties (= transversales) de votre système.
 Ces concepts sont souvent liés à plusieurs briques.
 Ils peuvent inclure de nombreux sujets différents, tels que
 
 * les modèles, particulièrement les modèles de domaine
-* les patrons d'architecture ou de conception
+* les modèles d'architecture ou de conception
```

Comment:
I'm ok with this fix but if we talk about "design pattern", in french, we usually say "patron de conception". But when we talk about architecture, "modèle" is the best in fact.